src/go/nerdctl-stub/generate/main_linux.go

```diff
@@ -1,6 +1,3 @@
-//go:build linux
-// +build linux
```

Comment:
Why is the `+build` line needed? Isn't it redundant now?

I thought running `go fix` would remove it, but it didn't, so maybe I'm missing something?

The same thing happens a couple more times below.

Reply:
Because the file has been renamed; `_linux` means the build tag is automatically added.  See [documentation](https://pkg.go.dev/cmd/go#hdr-Build_constraints); search for _If a file's name, after stripping the extension_

(Note that `_unix` does _not_ work because that's not a `GOOS`.)

Reply:
I still don't get it. If it has an implicit tag, why does it also need an explicit tag, especially if the tag is redundant with the `go:build` setting.

But I don't really care too much, as it isn't wrong anyways; I just don't understand why it is still being added.

Reply:
It _didn't_ have an implicit tag (it was just `main.go` before); because I renamed it, it no longer needs the explicit tag, hence the PR drops it.

Oh, wait, you mean the `+build` as opposed to the `//go:build`.  That hasn't been relevant for a while now; however, for backwards-compatibility reasons, the linters still get hung up on it, so we keep it to satisfy the linters.

Reply:
I'm sorry, I'm an idiot. For some reason I commented on one of the few cases where these lines were **removed** in the PR and not **added**.

But there are other cases where they were added, and you still added the `+build` one as well. I didn't realize the linters are still looking for them; that should be disabled in the linter rules, and the lines should be removed.

Anyways, a tiny thing, and not really important.

Reply:
No worries, we're just being a pair of idiots (me for failing to understand what you meant).

Re-reading the [documentation](https://pkg.go.dev/cmd/go#hdr-Build_constraints), thought, I think I might have it wrong:

> Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax. 

So it might be okay to drop it.  I'll do a check after.